bump_docker_base.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+#
+# This script bumps the version number pinned on the FROM line of a Dockerfile
+#
+# Author: wiha1292
```

Comment:
Not sure if it's important, but you could make this your GitHub user in the spirit of avoiding any CU-specific references.

Reply:
Good catch, thanks!